TODO: merge with glx-headers patch in patchset, similar
issues that seem to stem from Qt's GLX restoration.

https://bugs.gentoo.org/930107#c8
--- a/src/3rdparty/chromium/ui/gl/gl_display.h
+++ b/src/3rdparty/chromium/ui/gl/gl_display.h
@@ -12,4 +12,5 @@
 
 #include "ui/gl/gl_export.h"
+#include "ui/gl/gpu_preference.h"
 
 #if defined(USE_EGL)
